py3status/modules/xscreensaver.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+"""
+Toggle and lock Xscreensaver.
+
+This is a partial rewrite of dpms.py. Most credit goes to the original author
+Andre Doser. This module shows and controls the status of Xscreensaver
+
+This script is useful for people who let Xscreensaver manage dpms settings.
+Xscreensaver has its own dpms variables separate from xset. Dpms can be
+safely turned off in xset as long as xscreensaver is running.
+Settings can be managed using "xscreensaver-demo".
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module (default 60)
+    format: default format for this module (default '{output}')
+    string_off: display when Xscreensaver is not running (defaults '🖥️')
+    string_on: display when Xscreensaver is running (defaults '🖥️')
+
+ Format placeholders:
+    {output} display xscreensaver output
+
+@author neutronst4r <c7420{at}gmx{dot}net>
+"""
+from os import setpgrp
+from subprocess import Popen, DEVNULL, check_output, CalledProcessError
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    cache_timeout = 60
+    format = '{output}'
+    string_off = u'🖥️'
+    string_on = u'🖥️'
```

Comment:
`icon_...` would be more in keeping

also I'm not sure that this unicode char is very well supported.